lib/Zonemaster/Backend/Metrics.pm

```diff
@@ -0,0 +1,57 @@
+package Zonemaster::Backend::Metrics;
+
+use Log::Any qw($log);
+
+eval("use Net::Statsd");
```

Comment:
If not installed, nothing will happen. Correct?

Reply:
Yes that is correct, the feature will just be disabled.